make_wininsts.bat

```diff
@@ -1,10 +1,10 @@
-SET PATH=C:\Python26;%PATH%
```

Comment:
They are showing up as every line change in these diffs with no obvious change which tends to mean line endings.  It's not really a problem, just super thrashy / messes with git-blame.